hax/helper/configure.py

```diff
@@ -136,8 +136,9 @@ def get_join_ip() -> str:
                           env=self._get_pythonic_env())
 
         self.executor.run(Program([
-            'update-consul-conf', '--conf-dir', f'{self.conf_dir}',
-            '--kv-file', f'{self.conf_dir}/consul-kv.json'
+            'update-consul-conf', '--client', '--conf-dir',
```

Comment:
Don't we need to make `--client` be optional (i.e. controlled by some CLI argument)?